packages/admin/admin/src/Selected.tsx

```diff
@@ -24,7 +25,16 @@ const SelectEdit = (props: IProps) => {
         ) : (
             <Card>
                 <Box padding={4}>
-                    <Typography>Error :( {queryResult.error.toString()}</Typography>
+                    <Typography>
+                        <FormattedMessage
+                            id="cometAdmin.table.tableQuery.error"
```

Comment:
We may remove the "Error :(" text as well 😅

Reply:
should we only render the error message?

Reply:
Let's just keep it like that. Proper error handling is out of scope for this PR